gamemodes/terrortown/gamemode/server/sv_corpse.lua

```diff
@@ -490,6 +491,8 @@ function CORPSE.Create(ply, attacker, dmginfo)
         return
     end
 
+    rag:SetNWBool("real_player_corpse", realPlayerCorpse or false)
```

Comment:
do we need to network this at all? Only the server uses this if i'm not mistaken

besides that, we have `rag.player_ragdoll = true` a [few lines below](https://github.com/TTT-2/TTT2/pull/1577/files#diff-48172807435d5f9ba26d38fe099bdd5b56057fbb7ac6cbef676d68785c59877bR510)
wouldn't checking that in `FindCorpse` be enough?

Reply:
TTT2 only needs it on the server, BUt I thought it might be neat for addons such as defis to show why it fails to revive

Reply:
hmm i'm somewhat torn on this

it is indeed neat to provide this to addons, so that they don't have to sync this info themselves
but we would by default send every player what corpses are player corpses ...


i'd say if you want to make use of this in the near future (marker and/or necro defi i assume) i'm fine with it.
Otherwise i don't see a reason to include this "just in case"

Reply:
I feel like this is the same discussion as this: https://github.com/TTT-2/TTT2/issues/1386

I thought about adding it to the defi, but I'm not sure if I like adding it to the defi or if it would seem to OP

Reply:
you are right :D